data/templates/new_game_template

```diff
@@ -0,0 +1,18 @@
+[global]
+interactive_player="1"
+win_condition="scripting/win_conditions/endless_game.lua"
+peaceful="false"
+custom_starting_positions="false"
+map="maps/Lake_of_Tranquility.wmf"
+
+[player_1]
+tribe=
+init="headquarters.lua"
```

Comment:
Suggestion:

`starting_condition="headquarters"`

Reply:
I changed this now to support add-ons as well. Now you can either specify an add-on's internal name to use the starting condition defined by this add-on, or a filename to use one of the default starting conditions. Removing the `.lua` would make people wonder where these strings are defined, IMHO – with it (and with the new comment) it's clearer that this is a filename without path.

Reply:
ok :-)